samples/dnn/dnn_model_runner/dnn_conversion/ultralytics/yolov8_cv.py

```diff
@@ -0,0 +1,150 @@
+import os
+import os.path as osp
+import cv2
+import numpy as np
+from ultralytics.utils import ASSETS, yaml_load
+from ultralytics.utils.checks import check_yaml
+from ultralytics import YOLO
+
+CLASSES = yaml_load(check_yaml('coco128.yaml'))['names']
+
+COLOR_MAP = [
+    [113, 88, 194], [31, 102, 175], [11, 144, 50], 
+    [141, 86, 22], [102, 241, 115], [228, 113, 146], 
+    [180, 30, 22], [143, 97, 12], [186, 60, 46], [7, 46, 82], 
+    [151, 254, 142], [57, 80, 213], [56, 61, 251], 
+    [161, 120, 101], [20, 5, 65], [60, 102, 117], 
+    [219, 199, 218], [53, 213, 75], [1, 12, 111], 
+    [85, 82, 88], [140, 242, 149], [74, 71, 3], 
+    [12, 99, 105], [114, 51, 191], [196, 177, 240], 
+    [8, 38, 222], [45, 128, 142], [151, 70, 195], 
+    [57, 131, 15], [240, 79, 23], [1, 230, 181], 
+    [28, 240, 202], [93, 111, 75], [145, 105, 61], 
+    [106, 64, 6], [83, 134, 34], [206, 186, 185], 
+    [243, 153, 23], [23, 213, 140], [94, 144, 58], 
+    [217, 216, 229], [216, 202, 172], [151, 175, 132], 
+    [178, 101, 15], [57, 103, 139], [68, 118, 241], 
+    [249, 55, 244], [88, 63, 246], [106, 50, 177], 
+    [186, 253, 182], [69, 155, 170], [50, 155, 82], 
+    [128, 116, 235], [123, 83, 221], [241, 66, 44], 
+    [114, 68, 236], [116, 106, 36], [198, 152, 222], 
+    [64, 68, 122], [207, 34, 215], [83, 199, 140], 
+    [189, 214, 111], [214, 230, 107], [81, 248, 106], 
+    [37, 94, 186], [116, 15, 188], [66, 180, 87], 
+    [171, 200, 14], [205, 103, 83], [221, 92, 177], 
+    [93, 137, 220], [18, 79, 251], [88, 243, 155], 
+    [136, 208, 232], [121, 115, 245], [12, 213, 244], 
+    [174, 155, 25], [42, 47, 82], [56, 28, 90], [103, 33, 105]
+]
+
+
+class Opencv_Yolov8:
```

Comment:
There is https://github.com/opencv/opencv/blob/4.x/samples/dnn/object_detection.py. Can it be used?

Reply:
Yes. But if you want to make predictions quickly, Yolo model is a good option.  This sample code seems to use only fast-RCNN and SSD model, both of they are slower than Yolov8, even though they are good model.

Reply:
The sample supports YOLO models, take a look closer at https://github.com/opencv/opencv/blob/0507043a55e49069bf71b4ec629367237e03b723/samples/dnn/object_detection.py#L148

Reply:
Good question. Code sample worked only with Yolo from Darknet, not ONNX. Can you please show output layers of YOLOv8 using https://netron.app/?

Reply:
Here is the visualization of netron for yolov8 : 

![yolov8](https://github.com/opencv/opencv/assets/84863554/6a39ba52-a934-4df3-9fe9-77fa075e74b2)

output layer: float32[1,84,8400]

Please check, thanks!

Reply:
Thanks! So, there is no a unique layer type by which OpenCV can determine YOLO.

Can you please still try to fit your sample into https://github.com/opencv/opencv/blob/4.x/samples/dnn/object_detection.py but add an argument to run YOLO postprocessing?

Reply:
I understand that it is not a good idea but you can determine Yolo by check output Mat `size`.
Result is `1 x 25200 x 85` for Yolov5 and `1 x 84 x 8400` for Yolov8.

Reply:
@dkurt 
Hi, we have been committed the refactorized ```object_detection.py``` and ```model.yaml``` to make it enable 
to run yolov8 model under ONNX framework. 

Please check them, thanks.